speechbrain/augment/augmenter.py

```diff
@@ -438,20 +438,73 @@ def concatenate_outputs(self, augment_lst, augment_len_lst):
 
         return output, output_lengths
 
+    def replicate_multiple_labels(self, *args):
```

Comment:
This code is largely duplicated from `replicate_labels`, couldn't `replicate_labels` be made to call into this?

Reply:
Oui. Fixed.